packages/squiggle-lang/src/rescript/Utility/E.res

```diff
@@ -572,12 +572,22 @@ module A = {
       |> (x => Ok(x))
     }
 
-  let getByOpen = (a, op, bin) =>
-    switch getBy(a, r => bin(op(r))) {
-    | Some(r) => Some(op(r))
-    | None => None
+  let getByWithFn = (a, fn, boolCondition) => {
+    let i = ref(0)
+    let finalFunctionValue = ref(None)
+    let length = Belt.Array.length(a)
+
+    while i.contents < length && finalFunctionValue.contents == None {
+      let itemWithFnApplied = Belt.Array.getUnsafe(a, i.contents) |> fn
```

Comment:
(nitpick: I think we prefer `->` in the `CONTRIBUTING.md`, and for unary functions I don't think there's a difference)

Reply:
Good point, this was a mistake